### ISSUE TITLE
/data/data problem 

### ISSUE THREAD

**Issue by ag6ag1** [SimpleMobileTools/Simple-File-Manager#571]
hello, 

Thank you for the all apps! 

I'm 
![Uploading Screenshot_2022-04-19-06-24-39-010_com.simplemobiletools.filemanager.pro.jpg…]()
not able to see /data/data folders, only file manager package and google gms folders appears as in the below screenshot 

I've experience this problem on another file manager 
phone : xiaomi poco x3 pro 
Android 11 Miui 12 and Android 12 Miui 13

the same problem appears on another file manager (attached screenshot - axet file manager- but not seen in - material file manager -

ps: I'm using all your apps and hope I can enjoy this issue fixed

Thank you

**Comment by TheShanMan** [comment]
Yeah I'd like to see a fix for this too. I was just experiencing some frustration trying to look for an app's Android/data/<app>/files directory in my sd card and the <app> directory was showing as empty. I tried browsing to it via root but got the same thing there (in fact it switched from "root" to "sd card" as soon as I navigated into the sd card). I even tried showing hidden and that wasn't it either. But I can see it in the native files app and on my computer when I browse my phone via usb.

**Comment by MK73DS** [comment]
Same here, root access doesn't give the whole root permissions. I can see all of `/data/data` in Termux or even in an older version of Solid Explorer.
